sbot/timeout.py

```diff
@@ -0,0 +1,18 @@
+"""Code to kill robot after certain amount of time."""
+import logging
+from signal import SIGALRM, Signals, alarm, signal
+from types import FrameType
+
+LOGGER = logging.getLogger(__name__)
+
+
+def timeout_handler(signal_type: Signals, stack_frame: FrameType) -> None:
+    """Handle the `SIGALRM` to kill the current process."""
+    raise SystemExit("Timeout expired: Game Over!")
```

Comment:
I forget what the output of this message is? Would replacing this with a `print` and `exit(1)` work? Rather than printing a full stack trace, which may confuse competitors into thinking an error occurred.

Reply:
It just shows the message, like a `print`. At least during my testing of this feature it does.

Reply:
I think we wanted to use a `SystemExit` as it is harder to catch with a `try`.

Reply:
`exit` works by raising `SystemExit`, so I believe the code here is equivalent to `exit("Timeout expired: Game Over!")`.